packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -76,7 +76,8 @@ export default {
       // please note, these are the maximum bandwidth values
       // the server supports, minimums have to be tested
       audio: 64000,
-      video: 4000000
+      video: 4000000,
+      startBitrate: 2000
```

Comment:
just wondering if we actually need a separate value for this.. why not use the existing `video` value (divided by 1000)? also why 2000 and not 4000? what value did they test with?

if we are adding a new value then might be worth supporting a case where it's set to null or undefined and not appending `x-google-start-bitrate` in that case.

Reply:
The customer tested with 2000 , The audio video values are set in the TIAS line which was used for linus i am not sure if edonus is using it , Anyways those values are for the TIAS line which is to indicate the max bitrateValue supported , 

Its better to check with the edonus team on max bitrate